src/pages/layout/Search/SearchTransaction.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ResultLink from "./ResultLink";
+import {useGetTransaction} from "../../../api/hooks/useGetTransaction";
+
+type SearchTransactionProps = {
+  txnHashOrVersion: string;
+};
+
+export default function SearchTransaction({
+  txnHashOrVersion,
+}: SearchTransactionProps) {
+  const {isLoading, isError, data} = useGetTransaction(txnHashOrVersion);
```

Comment:
`useGetTransaction` doesnt return isLoading/isError https://github.com/aptos-labs/explorer/blob/main/src/api/hooks/useGetTransaction.ts - might be good to modify it to return it (see https://github.com/aptos-labs/explorer/blob/main/src/api/hooks/useGetAccountResource.ts)

Reply:
Sounds reasonable. Although I'm curious why we don't want to just return the full result. This way whoever uses this hook has the flexibility to select what fields are useful. Also see discussion here: https://github.com/aptos-labs/explorer/pull/85#discussion_r943964465.

Reply:
can do that too =) . just make sure it works also with [useGetAccountResource](https://github.com/aptos-labs/explorer/blob/main/src/api/hooks/useGetAccountResource.ts), as it has a different implementation than the others. also, need to fix it in here also https://github.com/aptos-labs/explorer/pull/146#discussion_r966524748

Reply:
now, both `useGetTransaction` and `useGetAccountResources` doesnt return `isLoading/isError/error` or anything else than `data`

Reply:
Sorry I'm not completely following. Do we want to make change to the return or this is looking good for now? Regardless, I feel that this issue worth a separate PR as this one is already pretty big and self-contained. Might worth it to address it in separate one.